x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -4,64 +4,107 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+
+import datemath from '@elastic/datemath';
 import expect from '@kbn/expect';
-import { indexBy } from 'lodash';
+import getRolledUpData from './rollup_info';
 
 export default function ({ getService, getPageObjects }) {
-  const esArchiver = getService('esArchiver');
+
   const log = getService('log');
-  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'header']);
+  const es = getService('es');
+  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'indexManagement', 'settings', 'discover']);
 
   describe('rollup job', function () {
-    this.tags('smoke');
-    before(async () => {
-      // init data
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await esArchiver.load('canvas/default');
+    const rollupJobName = 'rollup-to-be-' + Math.floor(Math.random() * 10000);
+    const indexName = 'rollup-to-be';
+    const now = new Date();
+    const pastDates = [
+      datemath.parse('now-1d', { forceNow: now }),
+      datemath.parse('now-2d', { forceNow: now }),
+      datemath.parse('now-3d', { forceNow: now }),
+    ];
+
+    async function loadDates(dates, prepend = 'to-be-rolled-up') {
+      for (const day of dates) {
+        await es.index({
+          index: `${prepend}-${day.format('MM-DD-YYYY')}`,
+          body: {
+            '@timestamp': day.toISOString(),
+            foo_metric: 1
+          }
+        });
+      }
+    }
+
+    it('create new rollup job', async () => {
+      const indexPattern = 'to-be*';
+      const interval = '1000ms';
+
+      await log.debug('make sure all dates have the same concept of "now"');
+      await log.debug(pastDates);
+      await loadDates(pastDates);
+
+      await PageObjects.common.navigateToApp('rollupJob');
+      await PageObjects.rollup.createNewRollUpJob(rollupJobName, indexPattern, indexName,
+        interval, ' ', true, { time: '*/10 * * * * ?', cron: true });
+
+      await PageObjects.common.navigateToApp('indexManagement');
+
+      await PageObjects.indexManagement.toggleRollupIndices();
+
+      await log.debug('If the index exists then the rollup job has successfully' +
```

Comment:
I think you want to call `getIndexList` list here and assert the rollup job us here?

Reply:
Strange. Musta accidentally deleted it.
It was there in previous commits. :)